src/ngAnimate/module.js

```diff
@@ -267,9 +267,22 @@
  * .message.ng-enter-prepare {
  *   opacity: 0;
  * }
- *
  * ```
  *
+ * ### Animating between subsequent value changes
+ *
+ * Sometimes you need to animate between multiple (2 and more), different expression states, whose values
```

Comment:
_2 and more_ --> _more than 2_

Reply:
Actually it will work also for animating between just 2 states, so IMO it should be 2 and more

Reply:
Fair enough. Since you can't animate between...one state, let's drop the `multiple (2 and more)` part.

> Sometimes you need to animate between different expression states whose values don't necessary need to be known or referenced in CSS styles.

Reply:
💯